jobs/carbon/templates/bin/carbon_ctl.erb

```diff
@@ -21,7 +21,7 @@ case $1 in
     # Set file permissions
     chown -RH vcap:vcap /var/vcap/sys/log/carbon
     chown -RH vcap:vcap /var/vcap/sys/run/carbon
-    chown -RH vcap:vcap /var/vcap/store/graphite
+    chown -H vcap:vcap /var/vcap/store/graphite
```

Comment:
Why is the `-H` left here.  From the chown manpage:

>      -H      If the -R option is specified, symbolic links on the command line are
>              followed.  (Symbolic links encountered in the tree traversal are not
>              followed.)

This suggests that it's meaningless when not combined with `-R`.